_posts/2016-06-09-configuring-aws-for-lambda.md

```diff
@@ -0,0 +1,139 @@
+
+  1. Click 'Create a new group'
+    {% img /assets/img/custom/blog/2016-06-09-configuring-aws-lambda/aws-04.png "" %}
+
+  1. Pick a name for the group
+    {% img /assets/img/custom/blog/2016-06-09-configuring-aws-lambda/aws-05.png "" %}
+
+  1. Click next, choose these policies:
+     * AWSLambdaFullAccess
+     * IAMFullAccess
+     * AmazonAPIGatewayAdministrator
+
+    {% img /assets/img/custom/blog/2016-06-09-configuring-aws-lambda/aws-06.png "" %}
+
+  1. Next. See this page: 
+    {% img /assets/img/custom/blog/2016-06-09-configuring-aws-lambda/aws-07.png "" %}
+
+  1. Click Create group
+    {% img /assets/img/custom/blog/2016-06-09-configuring-aws-lambda/aws-08.png "" %}
+
+  1. Find your new group
+    {% img /assets/img/custom/blog/2016-06-09-configuring-aws-lambda/aws-09.png "" %}
+
+  1. Click on 'Users' tab. It won't have any users, as you just created it.
+    {% img /assets/img/custom/blog/2016-06-09-configuring-aws-lambda/aws-10.png "" %}
+
+  1. Create new users: go to the menu on the left 'Users', click the button 'Create new users'
+    {% img /assets/img/custom/blog/2016-06-09-configuring-aws-lambda/aws-11.png "" %}
+
+  1. You need to generate as many users as you want. For a test, with one is enough. Pick a name for those users. In my case, 'test1_' 'and test2_'. Select 'Generate an access key for each user' 
+    {% img /assets/img/custom/blog/2016-06-09-configuring-aws-lambda/aws-12.png "" %}
+
+  1. Click 'Show User Security Credentials', to see the credentials. You'll use those to authenticate against AWS. 
+    {% img /assets/img/custom/blog/2016-06-09-configuring-aws-lambda/aws-13.png "" %}
+
+  1. In any case, it's better to download the credentials
+    {% img /assets/img/custom/blog/2016-06-09-configuring-aws-lambda/aws-14.png "" %}
+
+  1. A CSV with the tokens
+    {% img /assets/img/custom/blog/2016-06-09-configuring-aws-lambda/aws-15.png "" %}
+
+  1. Go to Groups, again
+    {% img /assets/img/custom/blog/2016-06-09-configuring-aws-lambda/aws-16.png "" %}
+
+  1. Click 'users' tab, then 'Add users to this group'
+    {% img /assets/img/custom/blog/2016-06-09-configuring-aws-lambda/aws-17.png "" %}
+
+  1. Find or filter the users that you want
+    {% img /assets/img/custom/blog/2016-06-09-configuring-aws-lambda/aws-18.png "" %}
+
+  1. Select them
+    {% img /assets/img/custom/blog/2016-06-09-configuring-aws-lambda/aws-19.png "" %}
+
+  1. Click 'Add Users'
+    {% img /assets/img/custom/blog/2016-06-09-configuring-aws-lambda/aws-20.png "" %}
+
+### Installing and installind the AWS CLI
+
+   1. Find the installer at [AWS CLI][aws-cli-installation]
+   1. Configure it, with this [help page][aws-cli-configuration]
+     * A sample of that
+
+    ````
+    $ aws configure
+    AWS Access Key ID [****************TEST]: AKIAJT6V4JYWUUS26WMQ
+    AWS Secret Access Key [****************TEST]: 2X0IH6NeEyQ2Ui8qQV5/p1w2ZM4bormRCtvly72T
+    Default region name [eu-west-1]: eu-west-1 #or any other zone
```

Comment:
I would obfuscate the key and id even if this one is disabled now. Doesn't give a good impression that we'd include secret information in the blog post.